src/screens/Bridge/Exchange-CrossChain/Exchange.tsx

```diff
@@ -145,7 +143,7 @@ function Exchange({ fetchExchangeTitle }: Props) {
   const currency = useFiatCurrency();
   const rate = getAssetRateInFiat(rates, toAsset?.address, currency);
 
-  const offersQuery = useOffersQuery(chain, fromAsset, toAsset, debouncedFromValue);
+  const offersQuery = useOffersQuery(chain, fromAsset, toAsset, fromValue);
```

Comment:
Is there a reason we're removing the debounced value? I thought we would want the delay so the sdk isn't called every time the user types a number (ie. if the user types 1000, the useOffersQuery hook will be called 4 times instead of once)